src/constants/colors/secondary.ts

```diff
@@ -6,48 +6,48 @@ const SECONDARY_COLORS = {
   // brand colors
   primary: '#4c0000',
   primaryTint1: '#b20000',
-  primaryTint2: null,
+  primaryTint2: '',
   primaryTint3: '#ff0000',
   secondary: '#ff8000',
-  tertiary: null,
+  tertiary: '',
```

Comment:
The `svg.fill` type is `string | undefined`, so `null` was causing problems. Adding an empty string means we maintain some styles like `border: 1px solid` (which defaults to black), whereas `undefined` would add the style `border: 1px solid undefined`, which does not apply.

This is all a TODO, anyway, but this is just an FYI.